docs/user/_sources/_static/params/seg_config.csv

```diff
@@ -8,4 +8,9 @@ seg_use_threshold,# Choose threshold option to further refine the resulting segm
 seg_WM_threshold_value,Set the threshold value for refining the resulting White Matter segmentation tissue mask.,"A float value, such as 0.95."
 seg_GM_threshold_value,Set the threshold value for refining the resulting Gray Matter segmentation tissue mask.,"A float value, such as 0.95."
 seg_use_erosion,Whether or not to use erosion to erode binarized tissue mask.,"Boolean, True or False (e.g., seg_use_erosion: True)."
-seg_erosion_prop,Erosion proportion, if use erosion to erode binarized tissue masks.,"A float value, such as 0.6."
\ No newline at end of file
+seg_erosion_prop,Erosion proportion, if use erosion to erode binarized tissue masks.,"A float value, such as 0.6."
+template_based_segmentation,If you choose template based segmentation, C-PAC does not segment anatomical images into White Matter, Gray Matter, and CSF any more. White Matter, Gray Matter, CSF masks will be generated based on template segmentation masks.,"String, 'EPI_template', 'T1_template', or 'None'. (e.g., template_based_segmentation: 'None')."
```

Comment:
Consider changing "C-PAC does not segment anatomical images into White Matter, Gray Matter, and CSF any more." to "C-PAC will not segment the anatomical images using a conventional tool."

Reply:
And then "Instead, White Matter, Gray Matter... " etc.